packages/polaris-viz/src/components/ChartErrorBoundary/ChartErrorBoundary.tsx

```diff
@@ -55,3 +63,22 @@ export class ChartErrorBoundary extends React.Component<
     return this.props.children;
   }
 }
+
+function checkForMismatchedData(data) {
+  if (data == null || data.length === 0) {
+    return;
+  }
+
+  const firstSetLength = data[0].data.length;
+
+  const hasMismatchedData = data.some(
+    (series) => series.data.length !== firstSetLength,
+  );
+
+  if (hasMismatchedData) {
+    console.warn(
```

Comment:
```suggestion
   // eslint-disable-next-line no-console
    console.warn(
```

Reply:
ESlint is complaining about the console log

<img width="1134" alt="image" src="https://user-images.githubusercontent.com/709616/187761510-e89e1979-2bc9-4dba-bead-b65a8cfa27fe.png">